test/upload/cli/test_config.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import unittest
+from argparse import Namespace
+
+import responses
+import boto3
+from moto import mock_s3
+
+from ... import CapturingIO, reset_tweak_changes
+from .. import mock_current_upload_area
+
+import hca
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
```

Comment:
The path hack on L15-16 is intended to make `hca` imports work by adding the `dcp-cli` directory to the module search path. Clearly it's not needed here since you `import hca` on the line directly above. Also, because this test is in a sub-sub-directory of `test`, you have one `'..'` too few.

Please also check for this in other test files in this directory.

Reply:
Nice catch.  Too much copy-pasta.